### PR TITLE
Background

### PR DESCRIPTION
**Pull request by maxx06**

### CODE REVIEW

File: src/components/schedule.js [hunk 1]
```diff
@@ -1,7 +1,8 @@
 export const Schedule = () => {
   return (
-    <section className='bg-accent2 w-screen text-white'>
-      Schedule Section
-    </section>
+    <div className='h-screen flex flex-col w-full text-white'>
+      <p>schedule</p>
+      <img src="/images/Shark.png" className='w-1/4 h-auto z-50 '/>
```
Comment (22raor) [inline src/components/schedule.js]: Failing build because this image doesn't have alt prop.

Comment (22raor) [reply src/components/schedule.js]: Fix that pls and make sure all other images have them too

File: tailwind.config.js [hunk 2]
```diff
@@ -3,14 +3,23 @@ module.exports = {
   content: ['./src/**/*.{js,jsx,ts,tsx}'],
   theme: {
     colors: {
-      primary: '#5A26A1',
```
Comment (arvindh-manian) [inline tailwind.config.js:6]: really like that y'all used the config 🙌

File: src/App.css [hunk 3]
```diff
@@ -36,3 +38,9 @@
     transform: rotate(360deg);
   }
 }
+
+
+.gradient-container {
```
Comment (arvindh-manian) [inline src/App.css:43]: in the future, can y'all comment why some choices are inconsistent (e.g., why some of the CSS is in tailwind classnames and some is in the css file)

not bad choices -- just important to document for the future ifl

**Review by arvindh-manian — APPROVED** [review]

**Pull request merged**